### PR TITLE
remove extra object creation from array dup

### PR DESCRIPTION
**Pull request by d-m-u**
https://github.com/kbrock/benchmark-sweet/pull/15 creates unnecessary objects and could be better organized

Screen Shot 2020-08-19 at 5 19 06 PM

### CODE REVIEW

File: lib/benchmark/sweet.rb [hunk 1]
```diff
@@ -89,37 +89,35 @@ def self.print_table(header_name, header_value, table_rows)
...
+
+    def self.standardize_size(arr, col_counts)
+      max_col_count = col_counts.max
+      col_names = arr.find { |row| row.count == max_col_count }.map(&:first)
+      arr.each do |line|
+        next if line.count == max_col_count
+
+        larr = line.to_a
+        full_line = col_names.select { |c| !larr.map(&:first).include?(c) }.flat_map { |col| larr.insert(col_names.index(col), [col, ""]) }
+        arr.map! { |l| l == line ? full_line.to_h : l }
```
Comment (kbrock) [inline lib/benchmark/sweet.rb:120]: using the bang in here is a little confusing.
maybe use a bang in the title, or use a regular map and assign arr (pirate) with the output of this method

I'm still a little confused why you need to do all this work

**Review by kbrock — COMMENTED** [review]
just a little confused as to why we are doing this

and the one trivial change which you don't have to do

**Comment by kbrock** [comment]
think we will regroup and fix the root of the problem so we don't have to do all this munging to fix up the data.

I think we'll pass in headers and change the interface here. But I'm leaving it to drew to make that call

**Comment by d-m-u** [comment]
yeah, you're right, sounds like the better plan to me
thanks

**Pull request closed without merging**